src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import {
+  Container, Col, Row,
+} from '@edx/paragon';
+
+import { CourseContext } from '../CourseContextProvider';
+import CourseSummaryCard from '../../executive-education-2u/components/CourseSummaryCard';
+import RegistrationSummaryCard from '../../executive-education-2u/components/RegistrationSummaryCard';
+import UserEnrollmentForm from '../../executive-education-2u/UserEnrollmentForm';
+import { useMinimalCourseMetadata } from '../data/hooks';
+
+const ExternalCourseEnrollment = () => {
+  const history = useHistory();
+  const { url } = useRouteMatch();
+  const {
+    state: {
+      courseEntitlementProductSku,
+    },
+  } = useContext(CourseContext);
+  const courseMetadata = useMinimalCourseMetadata();
+
+  const handleCheckoutSuccess = () => {
+    history.push(`${url}/complete`);
+  };
+
+  return (
+    <div className="fill-vertical-space page-light-bg">
```

Comment:
The mechanism by which we add the light bg color depicted in the [Figma](https://www.figma.com/file/p5FRghvTXVLZ8nqKx3TDwd/Unilever-flow?type=design&node-id=825-110940&t=z578ivIF6axqz15f-4) was refactored to be usable in any context, not just this one set of components.